add coq-msets-extra version 1.0.0

thtuerk opened the pull request:
First addition of MSetsExtra, an extension of the finite set library MSets, which focuses on efficient execution. This lib was presented at the Coq workshop at ITP 2016. Please see

https://github.com/thtuerk/MSetsExtra/blob/master/doc/mset.pdf

for high level documentation.

gares commented:
The package looks fine.  The only fishy thing is the "open ended" dependency on Coq.
We encourage conservative dependencies ( https://coq.inria.fr/opam/www/packaging.html ).
For example, if you package works with 8.4.5 and all 8.5.x, you should write `"coq" {>= "8.4.5" & < "8.6~"}`.  When 8.6 will be out, you may change the upper bound to "8.7~".

gares commented:
I also encourage you to give a look at ( https://github.com/coq/ceps/blob/master/text/003-opam-metadata.md ) and eventually add a `tags` field.

thtuerk commented:
@gares Many thanks for you quick and helpful comments. 0a4e2eb hopefully addresses the issues. I fixed the dependencies and added the `tag` field. Many thanks!

gares commented:
Awesome, thanks!
